### DISCUSSION TITLE
What is the health check endpoint in Renovate CE/EE?

### DISCUSSION THREAD

**Discussion by shenke-adsk** [mend/renovate-ce-ee#427] [Q&A] [answered]
In the old Renovate On-prem container, there is a health check endpoint: /status. Switching to Renovate CE, that endpoints seems broken.

**Comment by Gabriel-Ladzaretti** [comment, marked as answer]
Its GET /health
https://github.com/mend/renovate-ce-ee/blob/main/docs/api.md#health
